templates/shortcode-goal.php

```diff
@@ -1,10 +1,21 @@
 <?php
+
+use Give\Log\Log;
+
 /**
  * This template is used to display the goal with [give_goal]
  */
 
-$form_id          = get_the_ID(); // Form ID.
-$form        = new Give_Donate_Form( $form_id );
+/**
+ * @var int $form_id form id passed from the give_show_goal_progress() context
+ */
+
+if ( empty($form_id) ) {
+    Log::error('Error: Form ID can not be empty.');
```

Comment:
@JoshuaHungDinh Few notes on good logging:
1. You don't need to prefix it with "error" or anything, as you are already marking it as an error: `Log:error()`
2. Ask yourself if the description would help you to find the issue. Where is this happening? Would we know that?
3. If you have data to provide to the 'context' array then please do.
4. Make the log happen somehow so you can see what's actually recorded and make sure its enough and would really help. This is a favor to our future selves.

Reply:
Resolved in : https://github.com/impress-org/givewp/pull/6564/commits/9cf4b9c7
<img width="1567" alt="Screen Shot 2022-09-27 at 11 05 19 AM" src="https://user-images.githubusercontent.com/75056371/192603426-3edfd099-4fcc-4095-873c-560dae4066c7.png">